orangecontrib/educational/widgets/owkmeans.py

```diff
@@ -135,85 +137,71 @@ class OWKmeans(OWWidget):
     # other settings
     k_means = None
     autoPlaySpeed = settings.Setting(1)
-    lines_to_centroids = settings.Setting(0)
+    lines_to_centroids = settings.Setting(True)
     graph_name = 'scatter'
     outputName = "cluster"
-    button_labels = {"step1": "Reassign membership",
-                     "step2": "Recompute centroids",
-                     "step_back": "Step back",
-                     "autoplay_run": "Run",
-                     "autoplay_stop": "Stop",
-                     "random_centroids": "Randomize"}
+    STEP_BUTTONS = ["Reassign Membership", "Recompute Centroids"]
+    AUTOPLAY_BUTTONS = ["Run", "Stop"]
```

Comment:
I like that you've grouped the labels that you set in multiple places at a single place, so you don't have to hunt down the appearances of the same string. However, I think that this works better -- see the places where it's used.